packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/component.js

```diff
@@ -146,7 +146,9 @@ export function build_component(node, component_name, context, anchor = context.
 					return (
 						n.type === 'ExpressionTag' &&
 						n.expression.type !== 'Identifier' &&
-						n.expression.type !== 'MemberExpression'
+						n.expression.type !== 'MemberExpression' &&
+						n.expression.type !== 'UpdateExpression' &&
+						n.expression.type !== 'AssignmentExpression'
```

Comment:
what's the deal with these?

Reply:
We create deriveds of update and assignment expressions in the template which in turn was causing tests not to warn now that they have context and can see they are runes mode. I remove this and updated the `events` logic instead.